source/deimos/openssl/types.di

```diff
@@ -67,6 +67,9 @@ alias ASN1_ITEM_st ASN1_ITEM;
 struct asn1_pctx_st;
 alias asn1_pctx_st ASN1_PCTX;
 
+import deimos.openssl.buffer;
```

Comment:
I think the definition was in `types.di` to avoid a circular import.
It also matches what has been done for other types.